openstack/compute/v2/extensions/shelveunshelve/requests.go

```diff
@@ -0,0 +1,25 @@
+package shelveunshelve
+
+import "github.com/gophercloud/gophercloud"
+
+func actionURL(client *gophercloud.ServiceClient, id string) string {
```

Comment:
This should be moved to a new file called `urls.go`.

Reply:
This is how it is it currently done in some other extensions, do you want this to be a big refractor?

Reply:
I don't see how moving a function to a new file called `urls.go` is a big refactor. 

You're correct in that a few extensions are missing a `urls.go` file, but the vast majority of packages in this repo follow that pattern. I'd rather catch this inconsistency ahead of time.

If you don't want to, that's fine - I'll just do it later.

Reply:
I could do that as part of this PR, not a problem.

Reply:
Just to clarify: you'd only need to add `urls.go` for this new package - not fix up any of the other packages. That can be done during some general cleanup at another time.

Reply:
Since I've refactored it already, would you prefer to keep it this way or I can pull it out.

Reply:
Let's go ahead and keep it - thanks for doing that :)

Reply:
Cheers